cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/ExcludedBrokersForReplicaMoveTest.java

```diff
@@ -4,6 +4,7 @@
 
 package com.linkedin.kafka.cruisecontrol.analyzer;
 
+import com.linkedin.kafka.cruisecontrol.analyzer.goals.BrokerSetAwareGoal;
```

Comment:
Can we also update the following tests with the new goal?
* FixOfflineReplicaTest
* ExcludedBrokersForLeadershipTest
* RandomClusterTest
* RandomSelfHealingTest
* ReplicationFactorChangeTest
* SelfHealingWithOptimizedGoalTest

Reply:
- [ ] FixOfflineReplicaTest
- [x] ExcludedBrokersForLeadershipTest
- [ ] RandomClusterTest
- [ ] RandomSelfHealingTest
- [x] ReplicationFactorChangeTest
- [x] SelfHealingWithOptimizedGoalTest

Reply:
I am working on adding to these tests. Below tests have been updated -


- [x]     FixOfflineReplicaTest
- [x]     ExcludedBrokersForLeadershipTest
- [ ]     RandomClusterTest
- [ ]     RandomSelfHealingTest
- [x]     ReplicationFactorChangeTest
- [x]     SelfHealingWithOptimizedGoalTest

Reply:
I am working on adding to these tests. Below tests have been updated -


- [x]     FixOfflineReplicaTest
- [x]     ExcludedBrokersForLeadershipTest
- [ ]     RandomClusterTest
- [ ]     RandomSelfHealingTest
- [x]     ReplicationFactorChangeTest
- [x]     SelfHealingWithOptimizedGoalTest

Reply:
Yes, continuing to add more tests.